homework23_24.py

```diff
@@ -0,0 +1,113 @@
+# 1. Клік на "Complex app for testing - QA" в хедері переводить на домашню сторінку
```

Comment:
As you're adding real tests, lets keep a real project structure. `TestStartPage` class should be in `test_start_page.py` module. Modules like `homework`, `lesson` are left in past. This is one continues project.